Cod3rsGrowth.Servicos/Interfaces/IServicoVenda.cs

```diff
@@ -7,6 +7,7 @@ public interface IServicoVenda
         List<Venda> ObterTodos();
         void Criar(Venda venda);
         Venda ObterPorId(int IdDoItem);
+        int ObterNovoId();
```

Comment:
ObterNovoId não deve ficar dentro de serviço. Deve ser uma coisa do seu repositório. Imagina ele como um banco de dados. Então, quando for criar a adição do id deve acontecer dentro dele.